internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -1195,16 +1194,18 @@ func newAssistedCM(ctx context.Context, log logrus.FieldLogger, asc ASC) (client
 			"ENABLE_DATA_COLLECTION": "True",
 			"DATA_UPLOAD_ENDPOINT":   "https://console.redhat.com/api/ingress/v1/upload",
 
-			"NAMESPACE":       asc.namespace,
-			"INSTALL_INVOKER": "assisted-installer-operator",
+			"NAMESPACE":              asc.namespace,
+			"INSTALL_INVOKER":        "assisted-installer-operator",
+			"SKIP_CERT_VERIFICATION": "False",
 		}
-		// enable https only on OCP
+		// serve https only on OCP
```

Comment:
Sorry, for my understanding - is this comment still true if we're always enabling https here https://github.com/openshift/assisted-service/pull/6564/files#diff-c619bb36181b5e1dd76fb1fcc1f54068b16a8761f77106052fcbf5e2f5f9d356R1064

Reply:
Yeah, so the ingress will be https, but tls will terminate there. Meaning assisted-service will serve http between itself and the ingress controller.

That's why I changed this comment to say that we're enabling https in that the agent will expect https and we should provide the agent with certs to trust the ingress, but the assisted service itself is not serving https.

Reply:
Thanks for the explanation!